docs/contributing/control-plane/debug-gm.md

```diff
@@ -0,0 +1,52 @@
+This document is intended to provide contributors an introduction to develop [GM(GlobalManager)][framework] of the Sedna project. 
+
+### Debug GM
+1\. config GM:
+
+The config file is the yaml format:
+```yaml
+kubeConfig: ""
+namespace: ""
+websocket:
+  port: 9000
+localController:
+  server: http://localhost:9100
+```
+1. `kubeConfig`: kubernetes config file, default `""`
+1. `namespace`: the namespace GM watches, `""` means that gm watches all namespaces, default `""`.
+1. `websocket`: since the current limit of kubeedge(1.5), GM needs to build the websocket channel for communicating between GM and LCs.
+1. `localController`:
+   - `server`: to be injected into the worker to connect LC.
+
+Generate a config yaml:
+```shell
```

Comment:
It's recommend placing a blank line before and after code blocks to make sure rendered normally, such as in [m2r2](https://github.com/miyakogi/m2rdemo)

Reply:
better to be compatible with github markdown render engine. markdown users don't know the rule